.github/dependabot.yml

```diff
@@ -0,0 +1,31 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: "weekly"
+      time: "06:00"
+    open-pull-requests-limit: 5 
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: "weekly"
+      time: "06:02"
+    open-pull-requests-limit: 5 
+  - package-ecosystem: docker
+    directory: /cmd/mimir
+    schedule:
+      interval: "weekly"
+      time: "06:04"
+    open-pull-requests-limit: 5
+  - package-ecosystem: docker
+    directory: /mimir-build-image
+    schedule:
+      interval: "weekly"
+      time: "06:06"
+    open-pull-requests-limit: 5
```

Comment:
If we update the build image dockerfile shouldn't we also update the usages of the build image?

Reply:
for the moment to update the build image, we need to follow manual steps https://github.com/grafana/mimir/blob/8b8d822f39a4bb95189ef72cad9d2596f7225575/docs/internal/how-to-update-the-build-image.md?plain=1#L3, you are right, I think the best might be automate the build image update once some PR changed config there, build and push automatically. will remove this for the moment